.travis.yml

```diff
@@ -69,6 +69,16 @@ before_install:
 install:
 - if [ "x${PYCBC_CONTAINER}" == "xbuild_and_test" ] ; then
     travis_retry ./tools/install_travis.sh ;
+    touch ~/.ssh/id_rsa ~/.ssh/ldg_user ~/.ssh/ldg_token ;
```

Comment:
This won't work, as the travis secure environment variables are not available in pull requests.